website_cookiebot/models/res_config_settings.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Trey - Antonio González <antonio@trey.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    cookiebot_id = fields.Char(
+        string='Cookiebot ID',
+        related='website_id.cookiebot_id',
+        readonly=False,
+    )
+    has_cookiebot_id = fields.Boolean(
+        string='Cookiebot',
+    )
+
+    @api.onchange('has_cookiebot_id')
+    def onchange_has_cookiebot_id(self):
+        if not self.has_cookiebot_id:
+            self.cookiebot_id = False
```

Comment:
This field must be a computed inversable field instead:

```suggestion
    cookiebot_enabled = fields.Boolean(
        string='Use Cookiebot',
        compute="_compute_cookiebot_enabled",
        inverse="_invert_cookiebot_enabled",
    )
```

And, of course, add the respective compute and inverse methods.